docs/editor/accessibility.md

```diff
@@ -70,179 +70,181 @@ In `settings.json` file, nest the following code inside the outermost curly brac
 
->**Note**: For more information on finding complementary colors, go to [Adobe Color](https://color.adobe.com/create/color-accessibility) to access the color blind simulator and interactive color wheel.
+>**Note**: For more information on finding complementary colors, access the colorblind simulator and interactive color wheel at [Adobe Color](https://color.adobe.com/create/color-accessibility).
 
 ## Dim unfocused editors and terminals
 
-Unfocused views can be dimmed to make it clearer where typed input will go. This is especially useful when working with multiple editor groups or terminals. Enable this feature by setting `"accessibility.dimUnfocused.enabled": true`. You can control the dimness level with `accessibility.dimUnfocused.opacity`, which takes the opacity fraction from 0.2 to 1 (default 0.75).
+Unfocused views can be dimmed to make it more clear where typed input will go. This is especially useful when working with multiple editor groups or terminals. Turn this feature on by setting `"accessibility.dimUnfocused.enabled": true` and configure the dimness level with `accessibility.dimUnfocused.opacity`.
 
 ## Keyboard navigation
 
-You will find that VS Code provides an exhaustive list of commands in the **Command Palette** (`kb(workbench.action.showCommands)`) so that you can run VS Code without using the mouse.  Press `kb(workbench.action.showCommands)` then type a command name (for example 'git') to filter the list of commands.
+VS Code includes an exhaustive list of commands in the **Command Palette** (`kb(workbench.action.showCommands)`) so that you can use VS Code without a mouse.  Select `kb(workbench.action.showCommands)`, then type a command name (for example, "git") to filter the list of commands.
 
 VS Code also has many preset keyboard shortcuts for commands.
 
-![Keybindings for commands are displayed at the end of the command palette entry](images/accessibility/keyboard-shortcuts.png)
+![Key bindings for commands are displayed at the end of the command palette entry](images/accessibility/keyboard-shortcuts.png)
 
-You can also set your own keyboard shortcuts. **File** > **Preferences** > **Keyboard Shortcuts** (`kb(workbench.action.openGlobalKeybindings)`) brings up the Keyboard Shortcuts editor where you can discover and modify keybindings for VS Code actions. See [Key Bindings](/docs/getstarted/keybindings.md) for more details on customizing or adding your own keyboard shortcuts.
+You can also set your own keyboard shortcuts. **File** > **Preferences** > **Keyboard Shortcuts** (`kb(workbench.action.openGlobalKeybindings)`) opens the keyboard shortcuts editor, where you can discover and modify key bindings for VS Code actions. For more details on customizing or adding your own keyboard shortcuts, go to [Key Bindings for Visual Studio Code](/docs/getstarted/keybindings.md).
```

Comment:
We use "Keyboard Shortcuts editor" to more closely match the UI. Same for things like "Settings editor"